Add guidance around the number of sources a user should create (#4053)
segmentio/segment-docs: src/connections/sources/index.md

4. Enter a name for your source as well as any information on the setup page.
5. Click **Add Source**.

> > info ""

Reviewer: ```suggestion
> info "One source or multiple sources?"
```

> > **One source or multiple sources?** 

Reviewer: ```suggestion
```

> > Ultimately, it's up to you, but it really depends on how you want to organize your data. We suggest that you create one source for each type of data you want to collect. For example, a customer might have one source for all of their website tracking and a different source for any mobile tracking. We typically recommend having one source for each type of data (different libraries) for the following reasons:

Reviewer: ```suggestion
> Segment suggests that you create one source for each type of data you want to collect. For example, a you might have one source for all of their website tracking and a different source for any mobile tracking. Creating one source per data type provides the following benefits:
```

Reviewer: Important to remember that we're our customers and prospects are the main audience for our docs. We typically just use `you` as opposed to `a customer`.

Author: Thank you for pointing that out!
